pkg/module/manager.go

```diff
@@ -60,7 +69,16 @@ func (m *ManagerImpl) ExecuteModule(ctx context.Context, modName string, cfg hcl
 		return nil, fmt.Errorf("module not found %q", modName)
 	}
 
-	if err := mod.Configure(ctx, cfg, execReq.Params); err != nil {
```

Comment:
ill comment here as I can't comment on the ExecuteModule function, why we pass the name and config seperatly and then put providers and params in the request (i.e pass th config+ name there)

Reply:
fixed